RELEASES.md

```diff
@@ -0,0 +1,131 @@
+  interacting with the Moby API.
+
+## Support Horizon
+
+Support horizons will be defined corresponding to a release branch, identified
+by `<major>.<minor>`. Release branches will be in one of several states:
+
+- __*Future*__: An upcoming scheduled release.
+- __*Alpha*__: The next scheduled release on the main branch under active development.
+- __*Beta*__: The next scheduled release on the main branch under testing. Begins 2-4 weeks before a final release.
+- __*RC*__: The next scheduled release on the main branch under final testing and stabilization. Begins 1 week before a final release. For new releases where the source branch is main, the main branch will be in a feature freeze during this phase.
+- __*Active*__: The release is a stable branch which is currently supported and accepting patches.
+- __*Extended*__: The release branch is only accepting security patches.
+- __*End of Life*__: The release branch is no longer supported and no new patches will be accepted.
+
+Releases will be supported until at least the next _minor_ release. Any minor release
+may be supported longer if a maintainer elects to be the owner. The owners of a
+release branch may decide whether to support it as an active release accepting a wider
+range of bug fixes or as an extended security release which only receive security
+related fixes.
+
+### Current Releases
+
+| Main module | API     | Client | Status   | Release date | End of life | Owner(s) |
+|-------------|---------|--------|----------|--------------|-------------|----------|
+| v2.0        | v0.1.52 | v0.1.0 | Alpha    | 2025-08-dd   |             |          |
+| v2.1        | v1.x    | v1.x   | *Future* |              |             |          |
+
+### Pre-modules releases branches
+
+| Release Branch | Status      | End of Life  | Owner(s)         |
+|----------------|-------------|--------------|------------------|
+| 28.x           | Active      | v2.0 release | @moby-committers |
+| 27.x           | End of Life |              |                  |
+| 26.1           | End of Life |              |                  |
+| 26.0           | End of Life |              |                  |
+| 25.0           | Active      | See [mcr25]  | @corhere         |
+| 24.0           | End of Life |              |                  |
+| 23.0           | Extended    | See [mcr23]  | @corhere         |
+
+[mcr25]: https://docs.mirantis.com/mcr/25.0/compat-matrix/maintenance-lifecycle.html
+[mcr23]: https://docs.mirantis.com/mcr/23.0/compat-matrix/maintenance-lifecycle.html
+
+## API Stability
+
+The following table provides an overview of the components covered by
+Moby versions and the stability guarantees provided to users and importers.
+
+| Component        | Status        | Stabilized Version | Links |
+|------------------|---------------|--------------------|-------|
+| HTTP API         | Stable        | 1.0                | [Docker Engine API](https://docs.docker.com/reference/api/engine/) |
```

Comment:
We may need some wording about currently supported API versions outline that we dropped support for v1.23 and before (and may start moving the lower version).

In general (I added some wording in the API docs/README.md), support for legacy versions (especially really old ones) is "best effort"; some functionality may be missing when we downgrade the daemon or client to an older API version.

Reply:
I think in the above "Support Horizons" it would make sense to have a separate tables about API versions. I think its less confusing to have more tables with very specific information than trying to fit too much into one table. The current one there should probably only have the main module and we can separate the tables for client and API support and which versions they are related to.